docs/core/tools/global-tools.md

```diff
@@ -54,6 +54,8 @@ To install a tool as a global tool, use the `-g` or `--global` option of [dotnet
 dotnet tool install -g dotnetsay
 ```
 
+[!INCLUDE[](~/includes/dotnet-tool-install-arch-options.md)]
```

Comment:
I think it would be better to locate this note after the command output rather than between the command and its output.